script/deploy-from-docker.sh

```diff
@@ -30,11 +30,15 @@ function retry {
   return 0
```

Comment:
Not part of this PR, but for public consumption, this folder may be better off called something different and possibly containing its own README

Reply:
(Nevermind)